docs/blocks-stylesheet.md

```diff
@@ -61,47 +61,54 @@ The class name is generated using the block's name prefixed with `wp-block-`, re
 
 ## Enqueueing Editor-only Block Assets
 
-Like scripts, your block's editor-specific styles should be enqueued during the `enqueue_block_editor_assets` action.
+Like scripts, your block's editor-specific styles should be enqueued by assigning the `editor_styles` setting of the registered block type:
 
 ```php
 <?php
 
-function gutenberg_boilerplate_enqueue_block_editor_assets() {
-	wp_enqueue_script(
-		'gutenberg-boilerplate-es5-step02',
+function gutenberg_boilerplate_block() {
+	wp_register_script(
+		'gutenberg-boilerplate-es5-step02-editor-script',
```

Comment:
A seemingly basic question I wasn't sure of: Are handles required to be unique only between script and style registrations, or for both? i.e. Is it valid to:

```php
wp_register_script( 'foo', /* ... */ );
wp_register_style( 'foo', /* ... */ );
```

Reply:
They are free to be the same. I do it all the time, and core does it as well. For example, `customize-controls`:

* Script: https://github.com/WordPress/wordpress-develop/blob/53f1c74e1c55910f9b7e4aebc8033571d3e52df8/src/wp-includes/script-loader.php#L586
* Style: https://github.com/WordPress/wordpress-develop/blob/53f1c74e1c55910f9b7e4aebc8033571d3e52df8/src/wp-includes/script-loader.php#L1065

Reply:
>They are free to be the same.

Thanks! Updated in 5972c9b21ef3314c2cf91fc620de930bf97d860f.